traffic_ops/app/lib/API/Deliveryservice.pm

```diff
@@ -54,7 +54,10 @@ sub index {
 		$criteria{'me.logs_enabled'} = $logs_enabled ? 1 : 0;    # converts bool to 0|1
 	}
 
-	if ( !&is_privileged($self) ) {
+	my $tenant_utils = Utils::Tenant->new($self);
+	my $tenants_data = $tenant_utils->create_tenants_data_from_db();
+
+	if ( !&is_privileged($self) and !$tenant_utils->ignore_ds_users_table()) {
```

Comment:
The reason there are 2 separate parameters is that I thought that maybe the 2 features would be in use together. First of all, in the transition - lets say you migrated only part of the DSes to work with tenancy, you don't want the rest of the DSes not to operate according to the table.

Regarding the "ignore" vs. "use" - as the parameters are optional - does not exist on a clean installation - we need that the default of the variables will be false.